_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -2256,6 +2261,118 @@ inline void UpdateBRCParams(DdiTask  & task)
     GetHRDParamFromBRC(task, task.m_initCpbRemoval, task.m_initCpbRemovalOffset);
 }
 
+mfxStatus ImplementationAvc::CheckSliceSize(DdiTask &task, bool &bToRecode)
+{
+    mfxU32   bsSizeAvail = mfxU32(m_tmpBsBuf.size());
+    mfxU8    *pBS = &m_tmpBsBuf[0];
+    mfxStatus sts = MFX_ERR_NONE;
+    mfxExtCodingOption2    const & extOpt2 = GetExtBufferRef(m_video);
+
+    bToRecode = false;
+
+    MFX_CHECK(task.m_fieldPicFlag == 0, MFX_ERR_UNDEFINED_BEHAVIOR);
+
+    if ((sts = CopyBitstream(*m_core, m_video, task, task.m_fid[0], pBS, bsSizeAvail)) != MFX_ERR_NONE)
```

Comment:
I suggest to enable tracing here:
```
sts = CopyBitstream(*m_core, m_video, task, task.m_fid[0], pBS, bsSizeAvail);
MFX_CHECK(sts == MFX_ERR_NONE, Error(sts));
```

but I also prefer dropping ```Error``` debug wrapper and use ```MFX_CHECK_STS``` instead

Reply:
I can be done by separate commit for all status checks.

Reply:
Sure, I'm not insisting on changing it within this PR. But I think we really need to have it done